src/components/PoolRemove/index.tsx

```
@@ -1060,6 +776,26 @@ const PoolRemove = () => {
     }
   };
 
+  const debouncedSetFromAmount = useDebouncedCallback((value: string) => {
+    setFromAmount(value);
+  }, 300);
+
+  const handleClick = () => {
+    const amount = parseFloat(fromAmount);
+    if (isNaN(amount) || amount <= 0) {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)